server/queue/queue-rabbitmq/src/main/java/org/apache/james/queue/rabbitmq/view/cassandra/configuration/CassandraMailQueueViewConfiguration.java

```diff
@@ -70,13 +70,13 @@ public CassandraMailQueueViewConfiguration build() {
 
     public static CassandraMailQueueViewConfiguration from(Configuration configuration) {
         long sliceWindowInSecond = configuration.getLong(SLICE_WINDOW_PROPERTY_NAME);
-        Preconditions.checkState(sliceWindowInSecond > 1, "'sliceWindow' is a duration, then have to be positive");
+        Preconditions.checkState(sliceWindowInSecond >= MINIMAL_SLICE_WINDOW, "'sliceWindow' is a duration, then have to be positive");
```

Comment:
Not what I meant...

`Preconditions.checkState(sliceWindowInSecond >= MINIMAL_SLICE_WINDOW,  "'" + SLICE_WINDOW_PROPERTY_NAME '" is a duration, then have to be positive");`